aws/table_aws_ebs_volume.go

```diff
@@ -267,8 +275,8 @@ func getVolumeProductCodes(ctx context.Context, d *plugin.QueryData, h *plugin.H
 	return volumeAttributes, nil
 }
 
-func getEBSVolumeAkas(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
-	plugin.Logger(ctx).Trace("getEBSVolumeTurbotTags")
+func getEBSVolumeArn(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
```

Comment:
```suggestion
func getEBSVolumeARN(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
```

I think we want to capitalize any acronyms as per https://github.com/golang/go/wiki/CodeReviewComments#initialisms